plugins/slipping-away-users/frontend/public/localization/slipping.properties

```diff
@@ -1,17 +0,0 @@
-#slipping
```

Comment:
@ar2rsawseen Would changing the file name cause an issue for translations?

Reply:
actually there were two file names that were serving the same purpose (slipping-away-users.properties and slipping.properties). So I removed slipping.properties and merged its content inside slipping-away-users (this is also consistent with other int. naming convensions.